docs/pt/docs/help-fastapi.md

```diff
@@ -0,0 +1,148 @@
+# Ajuda FastAPI - Obter Ajuda
+
+Você gosta do **FastAPI**?
+
+Você gostaria de ajudar o FastAPI, outros usários, e o autor?
+
+Ou você gostaria de obeter ajuda relacionada ao **FastAPI**?
+
+Existem métodos muito simples de ajudar (A maioria das ajudas podem ser feitas com um ou dois cliques).
+
+E também existem vários modos de se conseguir ajuda.
+
+## Inscreva-se na newsletter
+
+Você pode se inscrever (na pouco frequente) [**FastAPI e amigos** newsletter](/newsletter/){.internal-link target=_blank} para receber atualizações:
+
+* Noticias sobre FastAPI e amigos 🚀
```

Comment:
* Notícias sobre FastAPI e amigos 🚀

Reply:
Hi @peidrao thanks for your review. Could you help me understand what is the problem here? Your review is the exaclty same phrase that the one i used.